src/api/spotify.ts

```diff
@@ -0,0 +1,55 @@
+import SpotifyWebAPI from 'spotify-web-api-node';
+
+export interface ISpotifyAlbumInfo {
+    id: string;
+    name: string;
+    image: string;
+    url: string;
+}
+
+export class SpotifyAPI {
+
+    private api: SpotifyWebAPI;
+
+    constructor(clientId: string, clientSecret: string) {
+        this.api = new SpotifyWebAPI({
+            clientId,
+            clientSecret,
+        });
+
+        // Retrieve an access token.
+        this.api.clientCredentialsGrant().then(
+            (data) => {
```

Comment:
Spacing is off

Reply:
imma be setting up eslint + prettier i think next, but good catch